checkout/rust/payment-customization/default/src/api.rs

```diff
@@ -3,29 +3,20 @@ use serde::{Deserialize, Serialize};
 pub type ID = String;
 
 #[derive(Clone, Debug, Deserialize)]
-pub struct Payload {
-    pub input: Input,
-    pub configuration: Config,
+#[serde(rename_all = "camelCase")]
```

Comment:
While you are in the file, let's conform all the function result attributes to be camel case. I think this is what we want to eventually recommend and enforce.